AUTHORS.txt

```diff
@@ -32,3 +32,4 @@ Mathieu Plante
 Michele Sardo
 Steven Siegl
 Benjamin Klebel-Knobloch
+Robert Eckelmann
```

Comment:
Don't forget to also add yourself to the second authors in the docs (docs/about/authors.rst)